isis/make/isismake.tsts

```diff
@@ -392,7 +392,7 @@ comparefiles: FORCE
 	      fi;                                                                  \
 	                                                                           \
 	      $(ISISROOT)/scripts/csvdiff.py $(TRUTH)/$$FILE $(OUTPUT)/$$FILE      \
-	          $$DIFFFILE >& /dev/null;                                         \
+	          $$DIFFFILE > /dev/null;                                          \
```

Comment:
This change looks reasonable to have left in 3.10 (but also doesn't seem like a huge deal to take out.) Did it come along with other commits that don't belong in 3.10, or did this break something?

Reply:
Never mind, I see that this (and the Jenkinsfile change) came in along with the the upgrade of geos in the `environment.yaml.`